src/Symfony/Component/Notifier/Bridge/GoogleChat/GoogleChatOptions.php

```diff
@@ -82,18 +81,41 @@ public function text(string $text): static
     }
 
     /**
+     * @param string $threadKey Opaque thread identifier string that can be specified to group messages into a single thread.
+     *                          If this is the first message with a given thread identifier, a new thread is created.
+     *                          Subsequent messages with the same thread identifier will be posted into the same thread.
+     *                          {@see https://developers.google.com/workspace/chat/api/reference/rest/v1/spaces.messages#thread}
+     *
      * @return $this
      */
-    public function setThreadKey(?string $threadKey): static
+    public function threadKey(string $threadKey): static
```

Comment:
`$threadKey` should be nullable to unset the value.

```suggestion
    public function threadKey(?string $threadKey): static
```

Reply:
This method no long exists.